packages/fxa-settings/src/lib/channels/helpers.ts

```diff
@@ -17,24 +17,8 @@ const REQUIRED_LOGIN_FIELDS: Array<keyof FxALoginRequest> = [
   'verified',
 ];
 
-export function notifyFirefoxOfLogin(
-  accountData: FxALoginRequest,
-  isSessionVerified: boolean
-) {
-  /* Only notify the browser of the login if the user 1) does not have to
```

Comment:
I originally copied this comment over from `fxa-content-server`'s `auth_brokers/fx-sync-channel.js`. @vpomerleau and I have been wondering whether or not we actually use the `/confirm` view - looks like prior to the settings refactor a couple years ago, we were sending users there in the scenario described in this comment. However, since then, we've been redirecting users to `signin_token_code` instead. 

Additionally, users don't have to have a verified session for this event when signing up through the Sync flow - I checked production, and as soon as signup is successful, the browser receives an update (which then says "Verify your account" or something of the sort with the email shown).

It's possible with the reset PW flow (only other flow currently using this function), we need the user to be verified. However, we have a regression ticket logged around this, FXA-8154. I wonder if previously we only tested and confirmed it was working when resetting a PW with a verified session without first "signing out" (which destroys the session), and that FXA-8154 may describe this signed-out case. I'll leave a comment in that ticket referring to this comment and we can add back some of the verified logic if needed.